PelotonEppSdkTests/BankAccountCreateTests.cs

```diff
@@ -17,10 +17,10 @@ public void TestCreateAccount()
             var createRequest = GetBasicRequest();
 
             // invent a new bank account token
-            createRequest.BankAccount.Token = Guid.NewGuid().ToString().Substring(0, 32);
+            createRequest.BankAccount.Token = Guid.NewGuid().ToString().Replace("-", "").Substring(0, 32);
```

Comment:
Consider .ToString("N") instead of Replace - it does exactly what you want and should have better performance